test/fixtures/multiple-onmessage-listeners-extension/content.js

```diff
@@ -1,13 +1,14 @@
 test("Multiple runtime.onmessage listeners which resolve to undefined", async (t) => {
   const res = await browser.runtime.sendMessage("test-multiple-onmessage-listeners:resolve-to-undefined");
 
-  if (navigator.userAgent.includes("Firefox/")) {
-    t.deepEqual(res, undefined, "Got an undefined value as expected");
-  } else {
+  if (navigator.userAgent.includes("Chrome/") && !navigator.userAgent.includes("Edge/")) {
```

Comment:
This is because Edge pretends to be Chrome, right?

The intent might be a bit clearer if you do UA sniffing in one location (e.g. test/fixtures/tape-standalone.js ) and then export the environment type to a variable (e.g. `TEST_BROWSER_NAME` or something like that). I'm using a variable instead of a property so that if you mistakenly use this global variable in a context where it is not defined, that a ReferenceError is triggered instead of getting the `undefined` value.